components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/GatewayArtifactsMgtDAO.java

```diff
@@ -451,6 +455,9 @@ public List<APIRuntimeArtifactDto> retrieveGatewayArtifactsByAPIIDs(List<String>
                             }
                         }
                         apiRuntimeArtifactDto.setFile(true);
+                        log.info("Adding runtime artifact dto for the API ID: " + apiRuntimeArtifactDto.getApiId()
```

Comment:
Lets make this a debug log.
I think for this one its okay to have one line each as we log multiple data.

Reply:
Fixed via https://github.com/wso2/carbon-apimgt/pull/11229/commits/c9cdba69f446b8698d33714d7ae49fdb3947ce33